app/imports/components/comments/Comments.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+
+import { Comment } from '/imports/components/comments/Comment.jsx';
+
+export const Comments = ({comments, commentsLimit, postId}) => {
+
+  if (comments === undefined) {
+    return null;
+  }
+
+  let commentsRows = [];
+  let moreComments = null;
+
+  if (commentsLimit === undefined ) {
```

Comment:
i don't think so, at least not yet. i would wait till we use the same param value again